modules/framework/galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/internal/streams/StreamsServiceImpl.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.framework.internal.streams;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import dev.galasa.framework.spi.ConfigurationPropertyStoreException;
+import dev.galasa.framework.spi.IConfigurationPropertyStoreService;
+import dev.galasa.framework.spi.streams.IStream;
+import dev.galasa.framework.spi.streams.IStreamsService;
+import dev.galasa.framework.spi.streams.StreamsException;
+
+public class StreamsServiceImpl implements IStreamsService {
+
+    private IConfigurationPropertyStoreService cpsService;
+    private static final String TEST_STREAM_PREFIX = "test.stream.";
+
+    public StreamsServiceImpl(IConfigurationPropertyStoreService configurationPropertyStoreService) {
+        this.cpsService = configurationPropertyStoreService;
+    }
+
+    @Override
+    public List<IStream> getStreams() throws StreamsException {
+        List<IStream> streamsList = new ArrayList<>();
+        try {
+            // Keys are in the form: test.stream.<stream-name>.property
+            String testStreamPrefix = "test.stream.";
```

Comment:
We have a TEST_STREAM_PREFIX constant we could use instead of redeclaring the prefix here?